apps/docs-snippets/src/guide/provider/querying-the-chain.test.ts

```diff
@@ -61,21 +56,38 @@ describe('querying the chain', () => {
       [100, assetIdA],
     ]);
 
-    const walletBalances = await wallet.getBalances();
-    // [
-    //   { amount: bn(42), assetId: baseAssetId }
-    //   { amount: bn(100), assetId: assetIdA }
-    // ]
-    // #endregion wallet-get-balances
+    const quantities: CoinQuantityLike[] = [
+      { amount: 32, assetId: baseAssetId, max: 42 },
+      { amount: 50, assetId: assetIdA },
+    ];
 
-    expect(walletBalances).toEqual([
-      { assetId: assetIdA, amount: bn(100) },
-      { assetId: baseAssetId, amount: bn(42) },
-    ]);
+    const utxoId = '0x00000000000000000000000000000000000000000000000000000000000000010001';
+    const messageNonce = '0x381de90750098776c71544527fd253412908dec3d07ce9a7367bd1ba975908a0';
+    const excludedIds: ExcludeResourcesOption = {
+      utxos: [utxoId],
+      messages: [messageNonce],
+    };
+
+    const spendableResources = await provider.getResourcesToSpend(
+      wallet.address,
+      quantities,
+      excludedIds
+    );
+
+    const tx = new ScriptTransactionRequest();
+    tx.addResources(spendableResources);
+    // #endregion get-spendable-resources-1
+
+    // #region get-spendable-resources-2
+    await wallet.getResourcesToSpend(spendableResources, excludedIds);
```

Comment:
Should we perform an assertion on the value returned here?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2408#discussion_r1647586635